packages/elements/src/chart/elements/chart.ts

```diff
@@ -0,0 +1,600 @@
+    }
+
+    return {
+      animation: {
+        duration: this.cssVarAsNumber('--animation-duration', '0')
+      },
+      elements: {
+        line: {
+          borderWidth: this.cssVarAsNumber('--line-width', '1'),
+          tension: this.cssVarAsNumber('--line-tension', '0.5')
+        }
+      },
+      plugins: {
+        tooltip: {
+          backgroundColor: this.getComputedVariable('--tooltip-background-color', 'transparent'),
+          titleColor: this.getComputedVariable('--tooltip-title-color', 'transparent'),
+          bodyColor: this.getComputedVariable('--tooltip-body-color', 'transparent'),
+          cornerRadius: this.cssVarAsNumber('--tooltip-border-radius', '0'),
+          caretSize: this.cssVarAsNumber('--tooltip-caret-size', '0'),
+          padding: {
+            x: this.cssVarAsNumber('--tooltip-padding-x', '--tooltip-padding', '0'),
+            y: this.cssVarAsNumber('--tooltip-padding-y', '--tooltip-padding', '0')
+          },
+          titleSpacing: this.cssVarAsNumber('--tooltip-title-spacing', '0'),
+          displayColors: false
+        },
+        legend: {
+          position: ['pie', 'doughnut'].includes(this.config?.type || '') ? 'right' : 'top',
+          labels: {
+            boxWidth,
+            boxHeight,
+            generateLabels: this.generateLegendLabels
+          }
+        }
+      }
+    };
+  }
+
+  /**
+   * Set global configuration of ChartJS
+   * @returns {void}
+   */
+  private setGlobalConfig (): void {
+    const cssStyle = getComputedStyle(this);
+
+    // Set font globals
+    ChartJS.defaults.color = cssStyle.getPropertyValue('color');
+    ChartJS.defaults.font.family = cssStyle.getPropertyValue('font-family');
+    ChartJS.defaults.font.size = Number(cssStyle.getPropertyValue('font-size').replace('px', ''));
+    ChartJS.defaults.font.style = cssStyle.getPropertyValue('font-style') as 'normal' | 'italic' | 'oblique' | 'initial' | 'inherit' | undefined;
+    // Set global grid color
+    ChartJS.defaults.scale.grid.color = (line) => {
+      return line.index === 0 ? this.getComputedVariable('--zero-line-color', 'transparent') : this.getComputedVariable('--grid-line-color', 'transparent');
+    };
+    if (this.config?.type === 'polarArea' || this.config?.type === 'radar') {
+      ChartJS.defaults.scales.radialLinear.ticks.showLabelBackdrop = false;
+    }
+  }
+
+  /**
+   * Handles a change of configuration object.
+   * This does not fire when a property of the config object changes,
+   * for this use this.updateChart() to apply changes.
+   * @returns {void}
+   */
+  protected onConfigChange (): void {
+    if (this.config) {
+      this.createChart();
+    }
+  }
+
+  /**
+   * Get as CSS variable and tries to convert it into a usable number
+   * @returns The value as a number, or, undefined if NaN.
+   */
+  protected cssVarAsNumber (...args: string[]): number | undefined {
+    const result = Number(this.getComputedVariable(...args).replace(/\D+$/, ''));
```

Comment:
Can you double check this security hotspot ? https://sonarcloud.io/project/security_hotspots?id=refinitiv_refinitiv-ui&pullRequest=687&resolved=false&types=SECURITY_HOTSPOT